Upgrade version number to 1.1.2

wenyongh opened the pull request.

loganek commented on RELEASE_NOTES.md:
```diff
@@ -12,6 +12,111 @@
+Create trap for error message when wasm_instance_new fails
+wamr-test-suites: Add support for ARM/RISCV by QEMU
+Fix zephyr sample build errors
+Fix source debugger error handling: continue executing when detached
```
I think this is more like a bugfix?

wenyongh replied:
Yes, sometimes it is not easy to classify whether it is bugfix or enhancement. I moved several items to "Bug Fixes".

loganek commented:
Great to see so many great changes since the last version! :confetti_ball:

wenyongh commented:
> Great to see so many great changes since the last version! 🎊

Yes, it is amazing. We will do more test and release the new version if all goes well.
